lib/pushmi_pullyu/cli.rb

```diff
@@ -140,11 +138,19 @@ def start_server_as_daemon
     require 'daemons'
 
     pwd = Dir.pwd # Current directory is changed during daemonization, so store it
-    Daemons.run_proc(config.process_name, dir: config.piddir,
-                                          dir_mode: :normal,
-                                          monitor: config.monitor,
-                                          ARGV: @options) do |*_argv|
 
+    options = {
+      ARGV:       @parsed_opts,
+      dir:        config.piddir,
+      dir_mode:   :normal,
+      monitor:    config.monitor,
+      log_output: true,
```

Comment:
With these log options I am forcing everything from the daemon to the log file specified (default is /log/pushmi_pullyu.log). Previously exceptions/etc were not being logged? (or if they were going to somewhere like `/var/log`)